README.md

```diff
@@ -195,23 +191,17 @@ struct WriteData: ParsableCommand {
                 .addTag(key: "type", value: "point-timestamp")
                 .addField(key: "value", value: .int(2))
                 .time(time: .date(Date()))
-        //
-        // Record defined as Tuple
-        //
-        let recordTuple: InfluxDBClient.Point.Tuple
-                = (measurement: "demo", tags: ["type": "tuple"], fields: ["value": .int(3)], time: nil)
-
-        let records: [Any] = [recordString, recordPoint, recordPointDate, recordTuple]
 
-        client.makeWriteAPI().writeRecords(records: records) { result, error in
+        client.makeWriteAPI().writePoints(points: [recordPoint, recordPointDate]) { result, error in
```

Comment:
Conventionally, the API should be `write(points:)` vs `writePoints(points:)` as the latter is read as "write points points"

Reply:
Fixed in https://github.com/influxdata/influxdb-client-swift/pull/25/commits/ccc88e33271c33382405d65da910186f817f4373